graphql-samples/mutations/collaboration/create-callout-on-collaboration

```diff
@@ -0,0 +1,21 @@
+mutation createCalloutOnCollaboration($data: CreateCalloutOnCollaborationInput!){
```

Comment:
This should be under `graphql-samples/mutations/create/create-callout-on-collaboration`